src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    
+    <div *ngIf="childrendataSource.data.length > 0 && !isLoading" class="table-wrapper">
+      <table mat-table [dataSource]="childrendataSource"  >
+        <tr hidden="true">
+          <th scope="col" >Child Name </th>
+          <th scope="col"  >DateOfBirth</th>
+          <th scope="col" >AfterBirthdayAge</th>
+          
+        </tr>
+        <ng-container matColumnDef="name">
+          <td mat-cell *matCellDef="let child">{{ child.name}} 
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="dateOfBirth">
+          <td mat-cell *matCellDef="let child">{{child.dateOfBirth | date:'dd-MM-YYYY' }} </td>
```

Comment:
```suggestion
          <td mat-cell *matCellDef="let child">{{child.dateOfBirth | date}} </td>
```
This will use the default date formatting for a persons geographic location/device settings